src/components/tab-bar/tab-bar.tsx

```diff
@@ -113,6 +123,25 @@ export class TabBar {
         this.changeTab.emit({ ...this.tabs[index], active: true });
     }
 
+    private handleScroll() {
+        const scrollLeft = this.scrollArea.scrollLeft;
+        const scrollRight = Math.floor(this.scrollContent.getBoundingClientRect().width -
+            this.scrollArea.getBoundingClientRect().width - scrollLeft);
+        console.log(`scrollLeft: ${scrollLeft}, scrollRight: ${scrollRight}`);
+
+        if (scrollLeft > 40) {
+            this.scrollerElement.classList.add('scroll-left');
+        } else {
+            this.scrollerElement.classList.remove('scroll-left');
+        }
+
+        if (scrollRight > 40) {
+            this.scrollerElement.classList.add('scroll-right');
+        } else {
+            this.scrollerElement.classList.remove('scroll-right');
+        }
```

Comment:
This circumvents Stencil's render-loop. I'm not sure if we want to do that, or if we would rather have these as State-properties, which trigger a re-render? Using State-props would be more consistent, but I'd be surprised if doing it this way wasn't more performant…

Thoughts?

Reply:
I think it's okay but wouldn't mind a second opinion from @jgroth

Reply:
I've changed this to use State-props instead in #1181, because we've also added the `disabled` attribute to the buttons in that PR.